### DISCUSSION TITLE
Multi Triggers for Popovers

### DISCUSSION THREAD

**Discussion by fuma-nama** [chakra-ui/chakra-ui#6969] [Q&A]
I know we can use multi-popovers with the same content instead.
However, since I use `Portal` to wrap my `PopoverContent`, it will cause about 20-50 portals to be rendered and mounted which may affect the performance.

How can I do that?

**Comment by ZiedHf** [comment]
I would first try isLazy and lazyBehavior.

**Reply by fuma-nama** [reply]
I can still see a lot of `<div class="chakra-portal">...</div>` on my devtools even if `isLazy` is enabled, is it a bug?
